pkg/render/node.go

```diff
@@ -167,6 +168,14 @@ func (c *nodeComponent) Objects() ([]client.Object, []client.Object) {
 		c.nodeRoleBinding(),
 	}
 
+	if c.cfg.Installation.KubernetesProvider == operatorv1.ProviderGKE {
```

Comment:
I think this should go in the core_controller where it is coming up with the components that need to be created. It should go before https://github.com/tigera/operator/blob/master/pkg/controller/installation/core_controller.go#L1045.
You can probably use a "render.NewPassthrough"

Reply:
Neat. I didn't know this existed. Will update the PR shortly. Thanks!